App.tsx

```diff
@@ -5,47 +5,96 @@ import { LoginScreen } from './src/features/login';
 import { ProfileScreen } from './src/features/profile';
 import { SettingsScreen } from './src/features/settings';
 import { SwiperScreen } from './src/features/swiper';
+import { MotivationListScreen } from './src/features/motivationList';
 
 import { TabBarIcon } from './src/components/tab-bar-icon';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
+import {
+  ApolloClient,
+  InMemoryCache,
+  ApolloProvider,
+  gql,
+} from '@apollo/client';
+
 const Tab = createBottomTabNavigator();
 
+const typeDefs = gql`
+  type Query {
+    page: Int!
+  }
+`;
+
+const cache = new InMemoryCache({
+  typePolicies: {
+    Query: {
+      fields: {
+        characters: {
+          keyArgs: false,
+          read(existing) {
+            return existing;
+          },
+          merge(existing = [], incoming) {
+            return [...existing, ...incoming.results];
+          },
+        },
+      },
+    },
+  },
+});
+
+const client = new ApolloClient({
+  uri: 'https://rickandmortyapi.com/graphql',
+  cache: cache,
+  typeDefs,
+});
```

Comment:
I would recommend to move the apollo config out of the App file (something like src/config/apollo/index.ts)

Reply:
good idea